roles/openshift_on_openstack/tasks/main.yml

```diff
@@ -158,6 +164,13 @@
     dest: "{{ all_yml_path }}"
     backup: yes
 
+- name: Copy {{ all_yml_path }} to the artifacts directory
+  synchronize:
+    src: "{{ all_yml_path }}"
+    dest: "{{ artifacts_directory }}"
+    mode: pull
+  ignore_errors: true
```

Comment:
Is the ignore_errors needed here or only in specific situations?

Reply:
I don't want the automation to fail if there is an error copying the file to the artifacts for any reason.